Formula/v8.rb

```diff
@@ -3,110 +3,90 @@
 class V8 < Formula
   desc "Google's JavaScript engine"
   homepage "https://github.com/v8/v8/wiki"
-  url "https://github.com/v8/v8-git-mirror/archive/5.1.281.47.tar.gz"
-  sha256 "63c9933227d6912689ea6bc012eea6a1fabaf526ac04bc245d9381e3ea238bf6"
-
-  bottle do
```

Comment:
Leave the bottle block be. CI will handle it.

Reply:
restored